commerce-service/src/main/java/com/liferay/commerce/internal/starter/ArgentinaCommerceRegionsStarter.java

```diff
@@ -36,52 +31,22 @@
 	property = "commerce.region.starter.key=" + ArgentinaCommerceRegionsStarter.ARGENTINA_NUMERIC_ISO_CODE,
 	service = CommerceRegionsStarter.class
 )
-public class ArgentinaCommerceRegionsStarter implements CommerceRegionsStarter {
+public class ArgentinaCommerceRegionsStarter
+	extends CommerceRegionsStarterBase implements CommerceRegionsStarter {
 
 	public static final int ARGENTINA_NUMERIC_ISO_CODE = 32;
 
-	public CommerceCountry getCommerceCountry(long companyId)
-		throws PortalException {
-
-		return _commerceCountryLocalService.fetchCommerceCountry(
-			companyId, ARGENTINA_NUMERIC_ISO_CODE);
-	}
-
-	public JSONArray getCommerceRegionsJSONArray() throws Exception {
-		Class<?> clazz = getClass();
-
-		String layoutsPath = "com/liferay/commerce/internal/argentina.json";
-
-		String regionsJSON = StringUtil.read(
-			clazz.getClassLoader(), layoutsPath, false);
-
-		return _jsonFactory.createJSONArray(regionsJSON);
-	}
-
 	@Override
 	public void start(ServiceContext serviceContext) throws Exception {
-		CommerceCountry commerceCountry = getCommerceCountry(
-			serviceContext.getCompanyId());
-
-		if (commerceCountry == null) {
-			return;
-		}
-
-		JSONArray jsonArray = getCommerceRegionsJSONArray();
-
-		for (int i = 0; i < jsonArray.length(); i++) {
-			JSONObject jsonObject = jsonArray.getJSONObject(i);
-
-			String code = jsonObject.getString("code");
-			String name = jsonObject.getString("name");
-			double priority = jsonObject.getDouble("priority");
-
-			_commerceRegionLocalService.addCommerceRegion(
-				commerceCountry.getCommerceCountryId(), name, code, priority,
-				true, serviceContext);
-		}
+		start(
+			_commerceCountryLocalService, _commerceRegionLocalService,
+			_jsonFactory, serviceContext, ARGENTINA_NUMERIC_ISO_CODE,
+			layoutsPath);
 	}
 
+	private static final String layoutsPath =
```

Comment:
@riccardo-alberti these `layoutsPath` fields should be `_LAYOUTS_PATH` -- that's our naming convention for private constants.